hazelcast/src/test/java/com/hazelcast/map/impl/mapstore/MapStoreTest.java

```diff
@@ -146,6 +146,34 @@ public Set<String> loadAllKeys() {
         assertFalse(loadCalled.get());
     }
 
+    @Test
+    public void map_set_does_not_load_old_value_when_offload_enabled() {
```

Comment:
I think it's better to test that "set does not load old value" here. Reasoning:
- "set does not load old value" is not specific to offload being enabled
-  force-offload builds will ensure the test is also executed with the offloaded code path.

Reply:
Maybe we can remove the word offload from test name but anyway by default we always enable offloading for all map-store related tests.

Reply:
Yes, I would remove "offload" from the test name and not explicitly set offloading enabled in the mapstore config.
This way we can have this test executed with the defaults (offloading enabled) and when we put in place not-offloadable-by-default test job we can get it to test also the non-offloadable code paths.

Reply:
done